docs/src/topics/SimpleCellValues_literate.jl

```diff
@@ -0,0 +1,95 @@
+# We start by including `Ferrite` and `Test`, 
+# to allow us to verify our implementation.
+using Ferrite, Test
+
+# Define a simple version of the cell values object, which only supports
+# identity mapping of scalar interpolations, without embedding. 
+struct SimpleCellValues{T, dim} <: Ferrite.AbstractCellValues
+    N::Matrix{T}             # Precalculated shape values
+    dNdξ::Matrix{Vec{dim,T}} # Precalculated shape gradients in the reference domain
+    dNdx::Matrix{Vec{dim,T}} # Cache for shape gradients in the real domain
+    M::Matrix{T}             # Precalculated geometric shape values
+    dMdξ::Matrix{Vec{dim,T}} # Precalculated geometric shape gradients
```

Comment:
I think we should expand here why these are matrices, as I am not confident that users will directly grasp why they should do this and what the matrix dimensions are.